src/classes/Passage.ts

```diff
@@ -193,10 +208,7 @@ export class Passage {
      */
     async getApp(): Promise<AppInfo> {
         try {
-            const configuration = apiConfiguration({
-                accessToken: this.#apiKey,
-            });
-            const client = new AppsApi(configuration);
+            const client = new AppsApi(this._apiConfiguration);
```

Comment:
this change would also be required in the magic link func

https://github.com/passageidentity/passage-node/blob/b76efe59ae3ea537aa29ef999e6a2d94282ee3bd/src/classes/Passage.ts#L185-L188

Reply:
fixed it